homeassistant/components/sensor/pushbullet.py

```diff
@@ -0,0 +1,120 @@
+"""
+PushBullet platform for sensor component.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.pushbullet/
+"""
+import logging
+import threading
+import voluptuous as vol
+
+from pushbullet import PushBullet
```

Comment:
You should only import objects from dependencies inside methods. Hence the requirement build step failing…